doc/backup.md

```diff
@@ -0,0 +1,116 @@
+# Database Backup and Restore Script
+
+## Overview
+
+This manual describes steps to perform for backup creation and its restoration.
+
+## Key Functions
+
+- **Backup**: Creates a timestamped `tar.gz` archive of the PostgreSQL database and Django media files.
+- **Restore**: Restores from a specified `tar.gz` backup archive.
+
+## Docker Compose Integration
+
+The `backup` service in `docker-compose.yaml` manages backup and restore using the `scripts/db.sh` script.
+
+### Configuration
+
+All variables can be set in the [environment file](.env.template). These include variables necessary for connection to the database, path to local folder where the backup is created and setup of cron tasks for backup.
+
+- **Volumes**:
+  - `${BACKUP_VOLUME:-../backups}:/backups`
+  - `.:/code`
+
+### Operations
+
+#### Enable Automatic Backups
+
+To ensure automatic backups are enabled, set `ENABLE_BACKUPS=true` (enabled by default):
+
+```bash
+ENABLE_BACKUPS=true docker compose up -d backup
```

Comment:
If a user already deployed the tool (`docker compose up -d`) this container is already running so starting it again is not needed.

Also, what is the purpose of having `backup` container running if we have already `scripts/backup_script.sh` being managed by OS level `crontab`?

Reply:
or are there like 3 ways to run backup?
a) manually by backup_script.sh
b) automatic - call backup_script.sh by cron
c) automatic - `backup` container running it from withing the container based on some schedule?

Reply:
ENABLE_BACKUPS: This was part of a work-in-progress documentation. It was initially used when we had cron jobs running inside the container, but it's no longer in use.

Reply:
we need the `backup` container to have the necessary libraries to do the db backups. the VM does not have the database.